flank-scripts/src/main/kotlin/flank/scripts/utils/File.kt

```diff
@@ -0,0 +1,21 @@
+package flank.scripts.utils
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+fun createSymbolicLink(
+    link: String,
+    target: String
+) {
+    Files.createSymbolicLink(
+        Paths.get(link)
+            .also { linkPath ->
+                if (Files.isSymbolicLink(linkPath))
+                    Files.delete(linkPath)
+            }
```

Comment:
I propose to make single line

Reply:
Doesn't make much difference for me, so I can agree.